dependabot.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      # Updates are scheduled at the beginning of the work week.
+      interval: "weekly"
+      day: "Monday"
+      time: "02:00"
+      # US Pacific Time
+      timezone: "America/Los_Angeles"
+    commit-message:
```

Comment:
Can we configure dependabot to add a "Signed off by" line to the commit since one of our lint steps in CI checks for this on each commit?

Reply:
It is unclear if dependabot will add the signed off line or not. The [documentation](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#commit-message) claims: "_Dependabot attempts to detect your commit message preferences_", but they also give you the option set a prefix. From what I can find, it looks like this is an outstanding [feature request](https://github.com/dependabot/dependabot-core/issues/1460), and [we will have to](https://github.com/dependabot/dependabot-core/issues/1460#issuecomment-545349408) waive the signoff requirement for dependabot PRs.

Reply:
However, I also see a [screenshot](https://github.com/dependabot/dependabot-core/issues/1683) from a dependabot commit that indicates we might OK. So maybe let's wait and see what a dependabot commit looks like and go from there?